README.md

```diff
@@ -205,45 +293,87 @@ This method returns a promise that resolves when all the promises in the chain r
 
 The parameter *series* is an array of promises and/or functions that return promises.
 
-For example, in the following code *p* rejects with value `"2"` in 2.5 seconds:
+The action function is triggered at the moment when the Promise instance is created. So using functions returning Promise instances to pass into `Promise.serial` makes instantiation sequential. I.e. a promise is created and the action is triggered only when the previous Promise in the series got resolved or rejected.
+
+For example, in the following code `p` rejects with value `"2"` in 2.5 seconds
```

Comment:
As written, the series doesn't reject - all promises resolve. The the value that p resolves with is "3" in 2.5 seconds.  

I added logs and measured time using harware.millis. Here are the logs I get when I run the example.

2018-08-17 13:09:30 -07:00 | [Device] | starting...
-- | -- | --
2018-08-17 13:09:30 -07:00 | [Device] | p1 - scheduling 1s timer
2018-08-17 13:09:30 -07:00 | [Device] | p3 - scheduling 0.5s timer
2018-08-17 13:09:31 -07:00 | [Device] | p3 0.5s timer expired
2018-08-17 13:09:31 -07:00 | [Device] | 0.508s from start
2018-08-17 13:09:31 -07:00 | [Device] | p1 1s timer expired
2018-08-17 13:09:31 -07:00 | [Device] | 1.012s from start
2018-08-17 13:09:31 -07:00 | [Device] | p2 - scheduling 1.5s timer
2018-08-17 13:09:33 -07:00 | [Device] | p2 1.5s timer expired
2018-08-17 13:09:33 -07:00 | [Device] | 2.528s from start
2018-08-17 13:09:33 -07:00 | [Device] | Resolved with value: 3
2018-08-17 13:09:33 -07:00 | [Device] | 2.535s from start

Reply:
yes, fixed. thanks!